app/assets/javascript/features.js

```diff
@@ -0,0 +1,730 @@
+/*
+ * Author = "Matthew D. Giammar"
+ * Email = "giammar.7@osu.edu"
+ */
+
+/* jshint esversion: 6 */
+
+/**
+ * Example structure
+ *
+ *  methods: [
+ *      {
+ *          "mth_0_rotor_frequency": [
+ *              5000,  // value
+ *              true,  // vary
+ *              null,  // expr
+ *              4900,  // min
+ *              5100,  // max
+ *              (the following are stored but not used)
+ *              null,  // brute_step
+ *              null,  // stderr
+ *              null,  // correl
+ *              5000,  // init_value
+ *              null,  // user_data
+ *          ],
+ *          "SP_0_operation_0_Scale_factor": [...],
+ *          ...
+ *          (more params)
+ *      },
+ *      ...
+ *      (more methods)
+ *  ]
+ */
+var paramGroups = {
+    modal_name: null,
+    spin_systems: [],
+    methods: [],
+};
+var doExternalIndexUpdate = true;
+const sysPrefix = ["sys"];
+const mthPrefix = ["mth", "SP"];
+const maxFeatureButtons = 8;
+
+
+/**
+ * Takes JSON representation of LMFIT Parameters object
+ * and groups Parameter objects into spin_systems and methods.
+ */
+var _reloadParamGroups = function () {
+    // console.log("_reloadParamGroups");
+
+    // If number of spin systems or methods is zero do not update
+    if (!(storeData.data.spin_systems.length && storeData.data.methods.length)) {
+        throw window.dash_clientside.PreventUpdate;
+    }
+
+    // Logic for creating params JSON if not already present
+    if (!storeData.data.params) {  // true if 'null', 'undefined' or empty str
+        document.getElementById("make-lmfit-params").click();
+        throw window.dash_clientside.PreventUpdate;
+    }
```

Comment:
The PreventUpdate will prevent all further updates. You only need to prevent the update of features UI. By using PreventUpdate you also prevent the SpinSystem UI.